test/inspector/test-inspector.js

```diff
@@ -151,6 +151,37 @@ function testInspectScope(session) {
   ]);
 }
 
+function testCommandLineAPI(session) {
+  console.log('[test]', 'Verify that command line API was injected');
```

Comment:
Same here - should I remove the `console.log` from the other functions in this file as well? Only added it for consistency with the existing code.